access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -380,8 +380,20 @@
       (context "/current-sids" []
         (OPTIONS "/" [] (common-routes/options-response))
 
-        (GET "/" {:keys [request-context params]}
-             (get-current-sids request-context params)))
+        (if (access-control-config/enable-sids-get)
+          (GET "/"
+               {:keys [request-context params]}
+               (do
+                 (error (format "client [%s] Using GET instead of POST when requesting current sids"
+                                (:client-id request-context)))
+                 (get-current-sids request-context params)))
+          {:status 404})
+
+        (POST "/"
+              {:keys [request-context body]}
+              (get-current-sids request-context (-> (slurp body)
+                                                    (json/parse-string true)
```

Comment:
Is keywordize-keys necessary if you use `json/parse-string true`?

Reply:
it was when I was developing, but I'll test again

Reply:
removed